packages/sdk-core/src/modules/database/BacktraceDatabase.ts

```diff
@@ -206,12 +213,19 @@ export class BacktraceDatabase implements BacktraceModule {
             const records = [...this._databaseRecordContext.getBucket(bucketIndex)];
 
             for (const record of records) {
+                if (!this.enabled) {
+                    return;
+                }
```

Comment:
can you change this to [throwIfAborted](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/throwIfAborted)? or, return some kind of information that the sending was aborted?

Reply:
good idea. Thx!